cos/templates/game.jinja2

```diff
@@ -1,33 +1,75 @@
 <!DOCTYPE html>
 <html lang="{{request.locale_name}}">
   <head>
+{#      <script type="text/javascript" src="{{request.static_url('cos:static/jquery-3.2.0.min.js')}}"></script>#}
       <!-- Bootstrap core CSS -->
-      <link href="//oss.maxcdn.com/libs/twitter-bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
 
       <!-- Custom styles for this scaffold -->
-      <link href="{{request.static_url('cos:static/sandstone.css')}}" rel="stylesheet">
+{#      <link href="{{request.static_url('cos:static/sandstone.css')}}" rel="stylesheet">#}
```

Comment:
These comments can be removed as they are not used

Reply:
Fixed